CONTRIBUTING.md

```diff
@@ -0,0 +1,258 @@
+- [Introduction](#introduction)
+- [The contribution process](#the-contribution-process)
+  - [Raise an issue](#raise-an-issue)
+  - [Create a fork](#create-a-fork)
+  - [Create a new folder](#create-a-new-folder)
+  - [Add license](#add-license)
+  - [Create a notebook](#create-a-notebook)
+  - [Commit new changes](#commit-new-changes)
+  - [Open a pull request](#open-a-pull-request)
+  - [Common recommendations for the review](#common-recommendations-for-the-review)
+- [CI/CD test passing guide](#cicd-test-passing-guide)
+  - [PEP 8 Style](#pep-8-style)
+  - [Notebook execution](#notebook-execution)
+  - [Format requirements](#format-requirements)
+- [Benchmarking result report](#benchmarking-result-report)
+
+## Introduction
+
+Thanks for considering a contribution to the MONAI Tutorials Repository and reading the Contributing Guidelines.
+
+The following is a set of guidelines for contributing tutorials to Project MONAI.
+We want to use the guideline to support the growth of the project, instead of restricting the depth and breadth of your contribution.
+If you have any questions about the guideline, please communicate with us and we are happy to discuss them with you.
+MONAI is an open-source project and its success relies on the entire community.
+
+Please feel free to propose changes to this document in a pull request(PR).
+
+## The contribution process
+
+### Raise an issue
+
+We encourage you to [raise an issue](https://github.com/Project-MONAI/tutorials/issues/new/choose) about your needs for MONAI tutorials.
+The idea may generate echo in the community and interest other contributors as well.
+
+### Create a fork
+
+When you are ready to kick off some coding, [create a new fork](https://github.com/Project-MONAI/tutorials/fork) of the `main` branch of this repository.
+
+After forking is complete, you can `git clone` the forked repo to your development environment, and use `git checkout -b` to create a new branch to freely experiment with your changes.
```

Comment:
Unless I've missed something I thought we should be having people checkout `dev` and branch from it rather than propose changes to `main`.